src/QMCWaveFunctions/tests/test_TrialWaveFunction_He.cpp

```diff
@@ -186,4 +186,189 @@ TEST_CASE("TrialWaveFunction flex_evaluateParameterDerivatives", "[wavefunction]
   CHECK(dhpsioverpsi2[0] == ValueApprox(dhpsi_over_psi_list.getValue(0, 1)));
 }
 
+
+UPtrVector<ParticleSet::ParticleGradient_t> create_particle_gradient(int nelec, int nentry)
+{
+  UPtrVector<ParticleSet::ParticleGradient_t> G_list;
+  for (int i = 0; i < nentry; i++)
+  {
+    G_list.push_back(std::make_unique<ParticleSet::ParticleGradient_t>());
```

Comment:
Can we use simply `emplace_back()` instead of `push_back(std::make_unique<ParticleSet::ParticleGradient_t>())`?

Reply:
Probably we can do `emplace_back(n)` and skip the following resize as well.